### ISSUE TITLE
"Engines of choice"?

### ISSUE THREAD

**Issue by Zireael07** [setzer22/blackjack#13]
Which engines are going to be supported? I must admit I'm intrigued, as the README makes it sound like we'll be able to change parameters in-game, instead of exporting static assets...

**Comment by setzer22** [comment]
> the README makes it sound like we'll be able to change parameters in-game, instead of exporting static assets...

Yes! That is very much the plan. My plan is to separate the blackjack "core" into a standalone library which you can embed into applications. The same files you create with the editor will then run on any program that is capable of calling rust code (and a C-like FFI layer will be exposed too to make this useful to a larger audience).

My plan is to make it so everyone can easily integrate blackjack into their game engine of choice, but I am planning to maintain an official integration for Godot, and I'm also interested in seeing integrations for rust engines like bevy or fyrox, so I will likely start something for those too
